rpc/src/v1/impls/parity.rs

```diff
@@ -46,7 +47,8 @@ use v1::types::{
 	BlockNumber, ConsensusCapability, VersionInfo,
 	OperationsInfo, ChainStatus,
 	AccountInfo, HwAccountInfo, RichHeader,
-	block_number_to_id
+	block_number_to_id,
+	H64 as RpcH64, H256 as RpcH256,
```

Comment:
Why was this double-import of `v1::types::H256` necessary? The `v1::types::` primitives have conversion facilities in place to convert to and from `ethereum_types::H*` (unrelated to this PR: I suspect that is not optimal to have different hash types) so I think it might work converting between them directly.

Reply:
It's my wrong. I have removed the double imported `v1::types::H256`.